cylc/flow/cycling/__init__.py

```diff
@@ -365,10 +365,9 @@ def set_offset(self, i_offset):
         """Deprecated: alter state to offset the entire sequence."""
         pass
 
-    @abstractmethod
     def is_on_sequence(self, point):
```

Comment:
```suggestion
    # NOTE: not using @abstractmethod because we need to
    # patch this method for caching purposes
    def is_on_sequence(self, point):
```